src/main/kotlin/org/cascadebot/cascadebot/utils/ChangeList.kt

```diff
@@ -0,0 +1,36 @@
+package org.cascadebot.cascadebot.utils
+
+import java.util.Collections
+
+class ChangeList<T>(val baseCollection: Set<T>) {
```

Comment:
Why do we need this?

Reply:
Used for aliases in the command info, it's so that we can still update global aliases even if people have edited. I'd recommend checking the code out in command info

Reply:
I see that it uses this, but don't see why this is used. Could you go through why we have a separate list class for handling this.

Reply:
I see that it uses this, but don't see why this is used. Could you go through why we have a separate list class for handling this. From the looks of it its letting you add and remove items but apply it later but for what purpose. I just don't want us to have classes we don't need so I want to make sure we need this.